contributors/design-proposals/kubelet-authorizer.md

```diff
@@ -146,29 +142,39 @@ for compatibility with deployment methods that bind other users or groups to tha
 
 ### Kubelets outside the `system:nodes` group
 
-Kubelets outside the `system:nodes` group would not be authorized by the node authorizer,
+Kubelets outside the `system:nodes` group would not be authorized by the `Node` authorization mode,
 and would need to continue to be authorized via whatever mechanism currently authorizes them.
 The node admission plugin would not restrict requests from these kubelets.
 
 ### Kubelets with undifferentiated usernames
 
 In some deployments, kubelets have credentials that place them in the `system:nodes` group,
 but do not identify the particular node they are associated with.
-Those kubelets would be broadly authorized by the node authorizer,
-but would not have secret and configmap requests restricted.
-The node admission plugin would not restrict requests from these kubelets.
+
+These kubelets would not be authorized by the `Node` authorization mode,
+and would need to continue to be authorized via whatever mechanism currently authorizes them.
+
+The node admission plugin *would* restrict requests from these kubelets.
+That plugin should not be used in a deployment with kubelets that do not identify themselves.
```

Comment:
After talking with @deads2k, I think it's worse to have an admission plugin that claims to restrict nodes, but actually is completely permissive with some nodes. A deployment that is ok with a mix of identifiable nodes and indistinguishable nodes, but also cares about segmenting what those nodes can do seems odd to me. I'd rather start by saying this is a tool for clusters that have identifiable nodes. We can always add an "enforce, but not really sometimes" mode if there is demand for it.